.repo-metadata.json

```diff
@@ -6,5 +6,6 @@
   "language": "python",
   "library_type": "INTEGRATION",
   "repo": "googleapis/python-bigquery-sqlalchemy",
-  "distribution_name": "pybigquery"
-}
\ No newline at end of file
+  "distribution_name": "pybigquery",
+  "api_id": "bigquery.googleapis.com"
```

Comment:
What does this do? :)

Reply:
Great question! I believe it is used by internal systems. After this change is made, I'm hoping that issues for `python-bigquery-sqlalchemy` are assigned to the `Data Analytics` team instead of `Yoshi Core` on the internal Yoshi Live dashboard.

Reply:
cc @tswast